tests/wpt-harness/expectations/FileAPI/blob/Blob-constructor.any.js.json

```diff
@@ -0,0 +1,191 @@
+{
+  "Blob interface object": {
+    "status": "FAIL"
```

Comment:
I'm not sure where should interface object be defined. This test fails on asserting the `Blob.length` which should be 0, but actual value is 1.

```
  assert_true("Blob" in globalThis, "globalThis should have a Blob property.");
  assert_equals(Blob.length, 0, "Blob.length should be 0.");
  assert_true(Blob instanceof Function, "Blob should be a function.");

```